test/dotnet-list-package.Tests/MSBuild.exe

```diff
@@ -0,0 +1 @@
+https://github.com/Microsoft/msbuild/issues/927
```

Comment:
Could we also add a comment as to why this is necessary, since the linked bug mentioned MSBuild APIs?